ProductsManegements/Product.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SimpleInventoryManagementSystem.ProductsManegements
+{
+    public class Product
+    {
+        private static int nextId = 1;
+        private string name = "";
+        private int quantity;
+        private decimal price;
+
+
+        public int Id { get; private set; }
+        public string Name
+        {
+
+            get { return name; }
```

Comment:
What's the value of implementing the setter and the getter here. 
you can just do:
```
        public string Name { get; set;}

```